book/src/SUMMARY.md

```diff
@@ -53,6 +53,12 @@
     * [MEV](./builders.md)
     * [Merge Migration](./merge-migration.md)
     * [Late Block Re-orgs](./late-block-re-orgs.md)
+* [Built-In Documentation](./help_general.md)
+    * [Beacon node](./help_bn.md)
+    * [Validator client](./help_vc.md)
+    * [Validator manager create](./help_vm_create.md)
+    * [Validator manager import](./help_vm_import.md)
+    * [Validator manager move](./help_vm_move.md)  
```

Comment:
Would be good to use capitals here so it's consistent with the other headings.

I also think we should split up the Validator Manager subcommands into their own headings.

So add a new general `Validator Manager` heading which contains the output of `lighthouse vm --help`, then 3 child headings titled `create`, `import` and `move` with their respective help texts.

Reply:
Add in [7075ddf](https://github.com/sigp/lighthouse/pull/4571/commits/7075ddf5e07f96e5294637be7ce71a3246e7ca6d)